nat-lab/tests/test_mesh_firewall.py

```diff
@@ -601,8 +604,8 @@ async def conntrack_on_stdout(stdout: str) -> None:
             connected_event,
         )
 
-        output_notifier.notify_output("FIN_WAIT", last_ack_event)
-        output_notifier.notify_output("CLOSE_WAIT", time_wait_event)
+        output_notifier.notify_output("LAST_ACK", last_ack_event)
+        output_notifier.notify_output("TIME_WAIT", time_wait_event)
```

Comment:
I'm curious about the reason for this change. I'm not against it, I would just like to hear the `why` behind it